app/src/main/java/com/example/parkingsystem_kotlin/mvp/model/ParkingModel.kt

```diff
@@ -1,11 +1,6 @@
 package com.example.parkingsystem_kotlin.mvp.model
 
 import com.example.parkingsystem_kotlin.mvp.contract.ParkingContract
-import com.example.parkingsystem_kotlin.util.Constants.PARKING_SIZE
+import com.example.parkingsystem_kotlin.util.Constants
 
-class ParkingModel: ParkingContract.Model {
-
-    private val parkingSize: Int = PARKING_SIZE
-
-    override fun getParkingSize(): Int = parkingSize
-}
+class ParkingModel(override var parkingSize: Int = Constants.PARKING_SIZE) : ParkingContract.Model
```

Comment:
please undo this change to keep the encapsulation

Reply:
done